### ISSUE TITLE
make all objects reference global app variable instead of a reference (Trac #27)

### ISSUE THREAD

**Issue by cfobel** [wheeler-microfluidics/microdrop#31]
Migrated from http://microfluidics.utoronto.ca/ticket/27
```json
{
    "status": "closed", 
    "changetime": "2014-04-17T19:39:01", 
    "description": "", 
    "reporter": "cfobel", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1397763541728826", 
    "component": "microdrop", 
    "summary": "make all objects reference global app variable instead of a reference", 
    "priority": "major", 
    "keywords": "", 
    "version": "0.1", 
    "time": "2012-01-04T03:27:08", 
    "milestone": "Microdrop 1.0", 
    "owner": "cfobel", 
    "type": "enhancement"
}
```

**Comment by cfobel** [comment]
Trac update at 20120104T07:14:51: 
* christian@fobel.net commented:

> commit 84f730888b6fad85d9f1dca8fdf7b4e6beccdde4
> Author: Christian Fobel <christian@fobel.net>
> Date:   Wed Jan 4 02:06:48 2012 -0500
> 
>     [SUBMODULE] Update dmf_control_board,video_reco...
>     
>     [SUBMODULE] Update dmf_control_board, video_recorder plugins.
>     Use app_context.get_app() to get App instance.  Fixes #31.


* christian@fobel.net changed resolution from "" to "fixed"
* christian@fobel.net changed status from "new" to "closed"

**Comment by cfobel** [comment]
Trac update at 20140417T19:39:01: ryan changed component from "core" to "microdrop"
